docs/book/user-guide/starter-guide/create-an-ml-pipeline.md

```diff
@@ -149,6 +149,24 @@ first_pipeline = first_pipeline.with_options(
 first_pipeline()
 ```
 
+## Configure with a yaml file
```

Comment:
Is this a duplicate of https://docs.zenml.io/user-guide/advanced-guide/configure-steps-pipelines#method-3-configuring-with-yaml? Do we need both?

Reply:
it is a light duplicate, here i is a light version just so people know about it. The other place in the docs is/can be more detailed